x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/40_ml_datafeed_crud.yml

```diff
@@ -151,9 +148,3 @@ setup:
               }
             }
           }
-
-  - do:
```

Comment:
This is so we at least have mixed cluster coverage on versions > 7.2.0. 

I could add this back, but then we would lose mixed cluster coverage for aggs for clusters containing nodes < 7.4.0